spec/lib/calculate/calculate_suggested_amounts_spec.rb

```diff
@@ -1,4 +1,5 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
+require 'rails_helper'
```

Comment:
This was missing

Reply:
🤯 Surprising how things work but seem like they should never work...